src/Iris.php

```diff
@@ -54,16 +53,32 @@ public function __construct()
         add_action('plugins_loaded', [__CLASS__, 'check_for_offload_media']);
 
         // IRIS :)
-        add_filter('wp_generate_attachment_metadata', [__CLASS__, 'iris_webp_converter'], 10, 3);
+        add_filter('wp_generate_attachment_metadata', [__CLASS__, 'irisWebpConverter'], 10, 3);
```

Comment:
Seeing as the class is `Iris` could we try to avoid adding `iris` to the beginning of associated methods? 

Am suggesting this mainly for code readability as there is enough content inferred from your method names already i.e. `Iris::webpConverter` is clear and concise as to its purpose, `Iris::irisWebpConverter` actually adds uncertainty in that it suggest there could be other adapters/methods within 'Iris' to handle conversions.

Hope that makes sense?

Reply:
Funnily enough I've done this already in recent work. You're totally right